docs/schema/clan.md

```diff
@@ -0,0 +1,2338 @@
+<!-- THIS FILE IS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT IT MANUALLY -->
+
+# Clan
+
+Clans are a permanent kind of party with additional features.
+
+A clan has a profil which includes several information like tag, name, description. The following use-case diagrams gives an overview about the topics which has to be handled with this schema:
+
+![](assets/20251218_215155_Clan_Hub_Client_Server.svg)
+
+![](assets/20251218_215852_CLan_Hub_Sever_Client.svg)
+
+### Manage clan
+
+Every user is allowed to create a clan with [create](#create). Then he is the clan leader. The clan leader is the only one who is allowed to delete the clan with the request [remove](#remove).
+
+Clan leader and clan co-leaders are able to the profile [update](#update).
+
+Clan leader and clan co-leaders are allowed to invite different players with [invite](#invite). (There are no limitations of the count of members.) The invitation can be cancelled by [cancelInvite](#cancelinvite). Invited players can accept invitiation by [acceptInvite](#acceptinvite) and decline by [declineInvite](#declineinvite).
+
+Only the clan-leader is able to kick members by [kickMember](#kickmember). Similarly, any member can leave the clan they are currently in with [leave](#leave).
+
+The clan leader is allowed to appoint up to three members as clan co-leader with [setRole](#setrole). The clan leader has also the possibility to overtake his leadership to a different member with the same command. (ATTENTION: By executing this command the clan leader loses his permissions!)
+
```

Comment:
I think "has also the possibility to transfer his leadership role to" is more correct.